lib/vanagon/platform/dsl.rb

```diff
@@ -228,6 +228,13 @@ def ssh_port(port = 22)
         @platform.ssh_port = port
       end
 
+      # Set the targetuser to use to connect
+      #
+      # @param type [String] user for the platform ('vagrant' for example)
+      def targetuser(user)
+        @platform.targetuser = user
+      end
+
```

Comment:
I think this is ok where it is. Several engines have the option of using a specific user.